presto-main/src/main/java/com/facebook/presto/execution/TaskExecutor.java

```diff
@@ -832,11 +858,13 @@ else if (t instanceof PrestoException) {
     {
         private final long startTime;
         private final String threadId;
+        private final Thread thread;
```

Comment:
Add a boolean field to this class so that we can print every thread that is over threshold.